quantify-from-bam.wdl

```diff
@@ -0,0 +1,90 @@
+import "tasks/stringtie.wdl" as stringtie_task
+import "tasks/biopet.wdl" as biopet
+import "tasks/htseq.wdl" as htseq
+
+workflow QuantifyFromBam {
+    File inputBam
+    File referenceGff
+    File referenceRefflat
+    Boolean? TPM
+    Boolean? FPKM
+    Boolean? fragmentsPerGene
+    Boolean? baseCountsPerGene
+
+    String sample
+    String strandedness
+
+    String outputDir
+
+    if (TPM || FPKM) {
```

Comment:
To make things simple we can do it maybe without the if statements? This means that all tools will always run, don't think this will be a big issue